### ISSUE TITLE
Subgraphx_overfitting

### ISSUE THREAD

**Issue by wangzeyu135798** [divelab/DIG#35]
Hi:
   In SubgraphX/models/train_gnn.py, you first trained a graph neural network for later calculation of each subgraph shapley value. While I use MUTAG dataset, run train_gnn.py, I found there is a overfitting problem. In this training process, MUTAG train accuracy is near to 100 percent and the gap between train accuracy and valid accuracy is large. I want to learn whether there occurs overfitting.

**Comment by Oceanusity** [comment]
Thank you for your issue. For the MUTAG dataset, we train the model using the whole dataset, and explain the example in the training dataset due to the accuracy in the training dataset is much higher. Because the goal of our task is to explain the important subgraphs for the predictions, it's fine to explain the training examples. And overfitting is not a main problem in the explainability task.
